forms/enrollment/naccid-enrollment-form_error_checks.csv

```diff
@@ -1,42 +1,63 @@
+enrl-c-034,034,Error,enrl,OLDADCID,Conformity,OLDADCID conforms,OLDADCID must be a valid code in list_of_adcids,Q6a. OLDADCID (Previous ADCID) must be a valid code in list_of_adcids,IF OLDADCID is not in list_of_adcids,n/a,n/a,No,n/a
+enrl-m-035,035,Error,enrl,NACCIDKNWN,Missingness,NACCIDKNWN conditionally present,If ENRLTYPE=2 (Transfer) then NACCIDKNWN cannot be blank,If Q1. ENRLTYPE (Assign NACCID or transfer?)=2 (Transfer) then Q7. NACCIDKNWN (Is NACCID known) cannot be blank,IF ENRLTYPE=2 and NACCIDKNWN=blank,n/a,n/a,Yes,n/a
+enrl-m-036,036,Error,enrl,NACCIDKNWN,Missingness,NACCIDKNWN conditionally absent,If ENRLTYPE=1 (Assign NACCID) then NACCIDKNWN must be blank,If Q1. ENRLTYPE (Assign NACCID or transfer?)=1 (Assign NACCID) then Q7. NACCIDKNWN (Is NACCID known?) must be blank,IF ENRLTYPE=1 and NACCIDKNWN not blank,n/a,n/a,Yes,n/a
+enrl-c-037,037,Error,enrl,NACCIDKNWN,Conformity,NACCIDKNWN conforms,"NACCIDKNWN must be an integer between 0 and 1, or 8 (Not a transfer)","Q7. NACCIDKNWN (Is NACCID known?) must be an integer between 0 and 1, or 8 (Not a transfer)",IF NACCIDKNWN<0 or (NACCIDKNWN>1 and ne 8),n/a,n/a,Yes,n/a
+enrl-m-038,038,Error,enrl,NACCID,Missingness,NACCID conditionally present,If NACCIDKNWN=1 (Yes) then NACCID cannot be blank,If Q7. NACCIDKNWN (Is NACCID known?)=1 (Yes) then Q7a. NACCID cannot be blank,IF NACCIDKNWN=1 and NACCID=blank,n/a,n/a,Yes,n/a
+enrl-m-039,039,Error,enrl,NACCID,Missingness,NACCID conditionally absent,If NACCIDKNWN=0 or 8 (No or Not a transfer) then NACCID must be blank,If Q7. NACCIDKNWN (Is NACCID known?)=0 or 8 (No or Not a transfer) then Q7a. NACCID must be blank,IF NACCIDKNWN in (0 or 8) and NACCID not blank,n/a,n/a,Yes,n/a
+enrl-m-040,040,Error,enrl,PTIDCONF,Missingness,PTIDCONF must be present,PTIDCONF cannot be blank,Q8. PTIDCONF (Confirm PTID) cannot be blank,IF PTIDCONF=blank,n/a,n/a,Yes,n/a
+enrl-p-1001,1001,Error,enrl,FRMDATE_ENRL,Plausibility,FRMDATE_ENRL earliest date allowed,FRMDATE_ENRL must not be from date before form was released,Header variable FRMDATE_ENRL must not be from date before form was released,IF FRMDATE_ENRL before (03-01-2024),n/a,n/a,Yes,n/a
+enrl-p-1002,1002,Error,enrl,PREVENRL,Plausibility,PREVENRL must align with ENRLTYPE,If ENRLTYPE=2 then PREVENRL must equal 1,If Q1. ENRLTYPE (Assign NACCID or transfer?)=2 (Transfer) then Q6. PREVENRL (Previously enrolled at an ADRC?) must be equal to 1 (Yes),IF ENRLTYPE=2 and PREVENRL in (0 or 9),n/a,"ENRLTYPE, PREVENRL",Yes,n/a
+enrl-p-1003,1003,Error,enrl,OLDADCID,Plausibility,OLDADCID cannot equal ADCID,OLDADCID cannot be the same as ADCID,Q6a. OLDADCID (Previous ADCID) cannot be the same as Header variable ADCID,IF OLDADCID =ADCID,n/a,"OLDADCID, ADCID",Yes,n/a
+enrl-p-1004,1004,Error,enrl,PTIDCONF,Plausibility,PTIDCONF cannot equal PTID,PTIDCONF cannot be the same as PTID,Q8. PTIDCONF (Confirm PTID) cannot be the same as Header variable PTID,IF PTIDCONF=PTID,n/a,"PTIDCONF, PTID",Yes,n/a

```

Comment:
I think the check should be PTIDCONF must be equal PTID ??

Reply:
Oops, yes, I completely reversed the language in my head. The error should be if they are not equal.